layout/pages/settings/online.xml

```diff
@@ -93,28 +93,11 @@
 
 				<Panel class="w-full flow-right">
 
-					<ModelPanel id="GhostModelPreview" class="settings__ghost" src="models/player/player_shape_base.mdl" mouse_rotate="true" antialias="true" />
+					<ModelPanel id="GhostModelPreview" class="settings__ghost" src="models/player/mom_player.mdl" mouse_rotate="true" antialias="true" />
 
 					<Panel class="w-fill flow-down">
-						<SettingsEnumDropDown text="#Settings_OnlinePlayer_Body" convar="mom_ghost_bodygroup" oninputsubmit="SettingsPage.onlineSettingsUpdateModel()" infomessage="#Settings_OnlinePlayer_Body_Info" tags="shape,model">
-							<Label text="#Settings_OnlinePlayer_Bodygroup_0" value="0" id="bodygroup0" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_1" value="1" id="bodygroup1" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_2" value="2" id="bodygroup2" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_3" value="3" id="bodygroup3" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_4" value="4" id="bodygroup4" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_5" value="5" id="bodygroup5" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_6" value="6" id="bodygroup6" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_7" value="7" id="bodygroup7" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_8" value="8" id="bodygroup8" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_9" value="9" id="bodygroup9" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_10" value="10" id="bodygroup10" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_11" value="11" id="bodygroup11" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_12" value="12" id="bodygroup12" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_13" value="13" id="bodygroup13" />
-							<Label text="#Settings_OnlinePlayer_Bodygroup_14" value="14" id="bodygroup14" />
-						</SettingsEnumDropDown>
-
-						<ConVarColorDisplay text="#Settings_OnlinePlayer_Color" convar="mom_ghost_color" oncolorchange="SettingsPage.onlineSettingsUpdateModel()" infomessage="#Settings_OnlinePlayer_Color_Info" tags="model,color" />
+						<ConVarColorDisplay text="#Settings_OnlinePlayer_PrimaryColor" convar="mom_player_color_primary" oncolorchange="SettingsPage.onlineSettingsUpdateModel()" infomessage="#Settings_OnlinePlayer_Color_Info" tags="model,color" />
+						<ConVarColorDisplay text="#Settings_OnlinePlayer_SecondaryColor" convar="mom_player_color_secondary" oncolorchange="SettingsPage.onlineSettingsUpdateModel()" infomessage="#Settings_OnlinePlayer_Color_Info" tags="model,color" />
```

Comment:
Actually, is the `infomessage` attr even used? Don't see a i18n string for it. Seems unnecessary anyway IMO.